<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetWidget</class>
 <widget class="QWidget" name="AssetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string>检测模式 [连续]</string>
   </property>
   <item>
    <property name="text">
     <string>检测模式 [连续]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>增益调节 [×1]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>音量调节 [×8]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黄色报警阈值[20]dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>红色报警阈值[20]dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>连续录波[5s]</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>最大值重置</string>
    </property>
   </item>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="headerDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>100</number>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white}</string>
   </property>
   <property name="text">
    <string>当前测量设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_current">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {font-family:WenQuanYi Micro Hei;font-size:25px;color:green}
background:transparent;color:gray;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>未设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_tips">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white}</string>
   </property>
   <property name="text">
    <string>设置已生效，仪器所有的测量数据都将归档到此设备档案下</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
